src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -0,0 +1,342 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Framework;
+using Mono.Cecil;
+using NUnit.Framework;
+using Xamarin.ProjectTools;
+
+namespace Xamarin.Android.Build.Tests
+{
+	[Category ("Node-1"), Category ("AOT")]
```

Comment:
I think `Node-1` is already taking the longest, should we put these in `Node-2` or `Node-3`?

Reply:
Good idea 👍, I was going to wait to see how timings looked after these changes run and try to do some rebalancing.

Reply:
`Node-2` seems most preferable here, the latest PR build shows the nunit3 test step on these legacy jobs taking anywhere from 30-43 minutes on Windows and 42-54 minutes on macOS.